src/pool/srv_pool.c

```diff
@@ -1025,6 +1097,9 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 	ds_cont_svc_step_up(svc->ps_cont_svc);
 	cont_svc_up = true;
 
+	rc = crt_register_event_cb(ds_pool_crt_event_cb, svc);
+	if (rc)
+		goto out;
```

Comment:
event_cb_registered = true? And on new line 1113:

    if (event_cb_registered)
        unregister();

Reply:
ah, ok.

Reply:
Thanks.